src/core/src/pass/serialize.cpp

```diff
@@ -1424,7 +1433,7 @@ bool pass::StreamSerialize::run_on_model(const std::shared_ptr<ov::Model>& model
 
     // Blobs
     hdr.consts_offset = m_stream.tellp();
-    std::string name = "net";
+    const std::string name = "net";
```

Comment:
make it const string view?

Reply:
It cannot be passed as reference to string below.